specs/pvs/corla.pvs

```diff
@@ -1499,7 +1619,9 @@ END authentication
 
 % @relation back_end inherit system_architecture_component
 % @relation front_end inherit system_architecture_component
-system_architecture_component: DATATYPE WITH SUBTYPES back_end, front_end
+system_architecture_component: DATATYPE
+  WITH SUBTYPES back_end,
+                front_end
 BEGIN
   IMPORTING information_systems, database, server
 
```

Comment:
line 1635/1757 "score" should be all caps "SCORE". We should be clear that the RLA tool in no way helps the Counties with the SCORE consistency check. Nor the other checks mentioned in this section if I understand correctly. So why are we including this information in this document?